astropy/table/tests/test_mixin.py

```diff
@@ -102,14 +103,44 @@ def test_io_ascii_write():
             t.write(out, format=fmt['Format'])
 
 
+def test_io_quantity_write(tmpdir):
+    """
+    Test that table with Quantity mixin column can be written by io.fits,
+    but not by io.votable and io.misc.hdf5. Validation of the output is done.
+    Test that io.fits writes a table containing Quantity mixin columns that can
+    be round-tripped (metadata unit).
+    """
+    t = QTable()
+    t['a'] = u.Quantity([1,2,4], unit='Angstrom')
+
+    filename = tmpdir.join("table-tmp").strpath
+    open(filename, 'w').close()
+
+    for fmt in ('fits', 'votable', 'hdf5'):
```

Comment:
I'd kept them in the test to ensure that writing to votable and hdf5 does in fact generate a ValueError, since in test_io_write_fail(mixin_cols), I am not checking for Quantity columns.